prereise/gather/data/remap_ba_area/remap_ba_area.py

```diff
@@ -0,0 +1,271 @@
+import json
+import os
+import shutil
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import pandas as pd
+import requests
+
+from prereise.gather.data.remap_ba_area.map_data import (
+    miso_name_map,
+    states_to_map_with_counties,
+    swpp_name_map,
+)
+
+
+def get_full_filename(file_name):
+    return os.path.join(os.path.dirname(__file__), file_name)
+
+
+def compose_ba_map(output_shape_file="remap_ba_area.shp"):
+    """
+    Create the control area and subregion map. For CAs heavily overlapped, we define several zones and match the CAs to
+    these zones. The ca_source name is either mapped to one zone in BA_map_name, or allocated to multiple zones in
+    BA_map_name with fractions based on the area.
+    The function generates the remap_ba_area shape file and remap_ba_area.tif file.
+
+    :param str output_shape_file: output shape file name
+    :return: (*geopandas.GeoDataFrame*)  -- the final control zone map
+    """
+
+    with open(get_full_filename("../BA_County_map.json")) as json_file:
+        ba2county_dict = json.load(json_file)
+    # Add two missing counties
+    # this may be fixed in future updates of BA_County_map.json
+    ba2county_dict["EPE"].append("Doña Ana__NM")
+    ba2county_dict["WAUE"].append("Oglala Lakota__SD")
```

Comment:
I'm pretty sure I remember this from the notebook that @merrielle and I originally put together. If we trust that these counties should be mapped like this, then we should update the `BA_County_map.json` file as part of this PR and remove these code lines.